### PR TITLE
include version number in desktop profile screen

### PR DESCRIPTION
**Pull request by rcullito**
Addresses #4267 

Display desktop version number (not pegged to mobile) on the user's profile screen next to the logout option.

For comparison with the design in Zeplin please see: https://app.zeplin.io/project/5a324ae5d60d197e7e3369cc/screen/5b1a62f7e14541167d97c7d3

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

**Review by maxhora — APPROVED** [review]
Probably, better to have single commit for those changes than 9 commits :)

File: src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs [hunk 1]
```diff
@@ -38,5 +40,7 @@
      [react/view {:style {:height 1 :background-color "#e8ebec" :margin-horizontal 16}}]
      [react/touchable-highlight {:on-press #(re-frame/dispatch [:logout])
                                  :style {:margin-top 60}}
-      [react/view
-       [react/text {:style {:color :red}} "Log out"]]]]))
+
+      [react/view {:style styles/logout-row}
+       [react/text {:style {:color :red}} "Log out"]
```
Comment (vitvly) [inline src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs]: Maybe it makes sense to use colors from https://github.com/status-im/status-react/blob/develop/src/status_im/ui/components/colors.cljs, as the shades for red are gray are different there.

**Review by vitvly — CHANGES_REQUESTED** [review]

**Comment by rcullito** [comment]
@MaxRis hah good point. shall rebase before sending this on.

File: src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs (hunk 1 above)
Comment (rcullito) [reply src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs]: nice catch! alright will pull those in. thanks @siphiuel

File: src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs [hunk 2]
```diff
@@ -38,5 +41,7 @@
      [react/view {:style {:height 1 :background-color "#e8ebec" :margin-horizontal 16}}]
      [react/touchable-highlight {:on-press #(re-frame/dispatch [:logout])
                                  :style {:margin-top 60}}
-      [react/view
-       [react/text {:style {:color :red}} "Log out"]]]]))
+
+      [react/view {:style styles/logout-row}
+       [react/text {:style {:color colors/red}} "Log out"]
```
Comment (vkjr) [inline src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs]: I think we should add i18n label instead of "Log out" here.

**Review by vkjr — APPROVED** [review]

**Review by vitvly — APPROVED** [review]

File: src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs (hunk 2 above)
Comment (rcullito) [reply src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs]: oh ok, nice catch @vkjr ! will add that in :+1:

**Comment by churik** [comment]
Checked (Mac OSx High Sierra): 
- installation, version is displayed.
- log in\log out

**Pull request merged**